include/cassandra.h

```diff
@@ -131,6 +131,11 @@ typedef struct CassString_ {
 #define CASS_INET_V6_LENGTH 16
 
 /**
+ * The size of an inet string including a null terminator.
+ */
+#define CASS_INET_STRING_LENGTH 46
```

Comment:
That's pretty cool. I'm going to keep the magic number because it follows the pattern of the other magic constants.